### PR TITLE
Add build instructions for the impatient

### PR DESCRIPTION
**Pull request by sebgoa**
**What this PR does / why we need it**:

* it adds a small section to build k8s for the impatient.
* it fixes a typo
* it removes the code-of-conduct file which was a redirect to the cncf code.

fixes part of #36655

### CODE REVIEW

**Comment by sebgoa** [comment]
the jenkins failure is not related.and we should be able to skip tests because this just touches a .md file.

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-## Kubernetes Community Code of Conduct
```
Comment (ixdy) [inline code-of-conduct.md]: why was this file deleted?

Comment (sebgoa) [inline code-of-conduct.md]: because we don't link to it from the main README anymore. We point directly to the CNCF code of conduct in the CNCF repo.

This was agreed in https://github.com/kubernetes/kubernetes/pull/36817

File: README.md [hunk 2]
```diff
@@ -50,14 +50,36 @@ Join a Special Interest Group ([SIG](https://github.com/kubernetes/community))
...
+### Building Kubernetes for the impatient
+
+If you want to build Kubernetes right away there are two options:
+
+* You have a working [Go environment](https://golang.org/doc/install).
+
+```
+$ go get k8s.io/kubernetes
+$ cd $GOPATH/src/k8s.io/kubernetes
+$ make
```
Comment (MHBauer) [inline README.md]: I am impressed and happy that this now works without having to manually `go get` more tooling.

I'm not sure if this is something to be fixed, but when I do the initial go get, I see some output:
```
$ go get k8s.io/kubernetes
package k8s.io/kubernetes: no buildable Go source files in /root/go/src/k8s.io/kubernetes
$
```
Not sure if we want to mention it, or find a way to clean it up. (Maybe it's understood that this will happen?)

Comment (sebgoa) [inline README.md]: I guess we could do `go get -d k8s.io/kubernetes` that should avoid the warning message as it won't try to build.

**Comment by ixdy** [comment]
@sebgoa the verify failure appears to be related to this PR:
```
I1209 08:01:56.703] Verifying hack/make-rules/../../hack/verify-munge-docs.sh
I1209 08:02:08.150] +++ [1209 08:02:08] Building the toolchain targets:
I1209 08:02:08.150]     k8s.io/kubernetes/hack/cmd/teststale
I1209 08:02:08.151]     k8s.io/kubernetes/vendor/github.com/jteeuwen/go-bindata/go-bindata
I1209 08:02:08.288] +++ [1209 08:02:08] Generating bindata:
I1209 08:02:08.289]     test/e2e/generated/gobindata_util.go
I1209 08:02:09.044] +++ [1209 08:02:09] Building go targets for linux/amd64:
I1209 08:02:09.044]     cmd/mungedocs
I1209 08:02:14.516] /go/src/k8s.io/kubernetes/docs/devel/community-expectations.md
I1209 08:02:14.516] ----
I1209 08:02:14.516] md-links:
I1209 08:02:14.517] not printing failed chunk: too many lines
I1209 08:02:14.517] On line 21: "../../code-of-conduct.md": target not found
I1209 08:02:14.517] On line 60: "../../code-of-conduct.md": target not found
I1209 08:02:14.517] 
W1209 08:02:14.669] FAIL: changes needed but not made due to --verify
I1209 08:02:14.670] /go/src/k8s.io/kubernetes/docs/ is out of date. Please run hack/update-munge-docs.sh
I1209 08:02:14.672] FAILED   hack/make-rules/../../hack/verify-munge-docs.sh	18s
```

**Comment by ixdy** [comment]
... though if you try to make changes to `docs/devel/community-expectations.md`, the bot will get mad, since changes to `docs/devel` are now disallowed.

**Comment by ixdy** [comment]
Might need to leave `code-of-conduct.md` in the repo until https://github.com/kubernetes/kubernetes/issues/37827 is fixed.

**Comment by sebgoa** [comment]
ok reverted the removal of the code of conduct.
Added a '-d' option to the go get line but it does not remove the warning.
